### ISSUE TITLE
Taxon core extensions for list types #81

### ISSUE THREAD

**Issue by djtfmartin** [AtlasOfLivingAustralia/species-lists#57]
Certain categories of lists should eb published using a developed darwin core Taxon extension. This would include

1.  Sensitive lists 
2. Conservation lists 
3. Invasive lists. 

These extensions will include vocabularies where appropriate. The List tool should perform vaildation on these lists, rejecting lists with the specified category if fields are missing, values dont adhere to controlled vocabs etc.

**Comment by peggynewman** [comment]
Wouldn't this be a taxon core?

**Comment by djtfmartin** [comment]
Yes, but i think we need to define extensions to taxon core for these list types (unless we can reuse existing extensions/terms).

**Comment by TaniaGLaity** [comment]
Lists Project Team and broader ALA need to decide on Taxon Core Extensions and develop vocabs in consultation
